walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.replaceAll("[^a-zA-Z0-9]", "").trim();
```

Comment:
This is cool that your code will also handle spaces in `name`. Mind adding a test to verify that that logic path works?

Reply:
Sure, will do.

Reply:
Ah sorry - should have been more specific - I meant that I think this will handle `Ada Lovelace` correctly as input. Mind adding that test?